java/elemental2/dom/integer_entities.txt

```diff
@@ -4,6 +4,9 @@ elemental2.dom.Blob.slice.start
 elemental2.dom.ByteLengthQueuingStrategy.size
 elemental2.dom.ByteLengthQueuingStrategy.SizeChunkType.getByteLength
 elemental2.dom.ByteLengthQueuingStrategy.SizeChunkType.setByteLength.byteLength
+elemental2.dom.CloseEventInit.getCode
+elemental2.dom.CloseEventInit.setCode.code
+elemental2.dom.CloseEvent.code
```

Comment:
elemental2.dom.CloseEventInit and elemental2.dom.CloseEvent do not exist.

Reply:
Oops- I was waiting for a closure-compiler PR to land and thought it had overnight. I will ping this issue when it lands - sorry for the noise

Reply:
The commit landed in https://github.com/google/closure-compiler/commit/a6b5dd3b5b834d3734db10c613eddfceff94559e so this branch should be good to go now. Thanks